docs/en/docs/sql-manual/sql-reference/Data-Types/JSONB.md

```diff
@@ -35,8 +35,775 @@ under the License.
+|   18 | {"k1":"v31","k2":300,"a1":[{"k1":"v41","k2":400},1,"a",3.14]} | object               |
+|   26 |                                          {"k1":"v1","k2":200} | object               |
++------+---------------------------------------------------------------+----------------------+
+19 rows in set (0.02 sec)
+
+mysql> select id, j, jsonb_type(j, '$.k1') from test_jsonb order by id;
++------+---------------------------------------------------------------+-------------------------+
+| id   | j                                                             | jsonb_type(`j`, '$.k1') |
++------+---------------------------------------------------------------+-------------------------+
+|    1 |                                                          NULL | NULL                    |
+|    2 |                                                          null | NULL                    |
+|    3 |                                                          true | NULL                    |
+|    4 |                                                         false | NULL                    |
+|    5 |                                                           100 | NULL                    |
+|    6 |                                                         10000 | NULL                    |
+|    7 |                                                    1000000000 | NULL                    |
+|    8 |                                           1152921504606846976 | NULL                    |
+|    9 |                                                          6.18 | NULL                    |
+|   10 |                                                        "abcd" | NULL                    |
+|   11 |                                                            {} | NULL                    |
+|   12 |                                         {"k1":"v31","k2":300} | string                  |
+|   13 |                                                            [] | NULL                    |
+|   14 |                                                     [123,456] | NULL                    |
+|   15 |                                                 ["abc","def"] | NULL                    |
+|   16 |                              [null,true,false,100,6.18,"abc"] | NULL                    |
+|   17 |                            [{"k1":"v41","k2":400},1,"a",3.14] | NULL                    |
+|   18 | {"k1":"v31","k2":300,"a1":[{"k1":"v41","k2":400},1,"a",3.14]} | string                  |
+|   26 |                                          {"k1":"v1","k2":200} | string                  |
++------+---------------------------------------------------------------+-------------------------+
+19 rows in set (0.03 sec)
+
+```
 
 ### keywords
+JSONB, JSON, jsonb_parse, jsonb_parse_error_to_null, jsonb_parse_error_to_value, jsonb_extract, jsonb_extract_isnull, jsonb_extract_bool, jsonb_extract_int, jsonb_extract_bigint, jsonb_extract_double, jsonb_extract_string, jsonb_exists_path, jsonb_type
```

Comment:
only need `JSONB, JSON`

Other functions has their own docs

Reply:
the examples are useful for other functions and referenced by other docs